cookies/resources/testharness-helpers.js

```diff
@@ -0,0 +1,42 @@
+// Given an array of potentially asynchronous tests, this function will execute
+// each in serial, ensuring that one and only one test is executing at a time.
+//
+// The test array should look like this:
+//
+//
+//     var tests = [
+//       [
+//         "Test description goes here.",
+//         function () {
+//           // Test code goes here. `this` is bound to the test object.
+//         }
+//       ],
+//       ...
+//     ];
+//
+// The |setup| and |teardown| arguments are functions which are executed before
+// and after each test, respectively.
+function executeTestsSerially(testList, setup, teardown) {
```

Comment:
@jgraham: It would be helpful if you could take a look at this to help me understand whether there's a more idiomatic way of accomplishing this goal. Ideally, we could put something like this into `testharness.js` proper, and avoid the helper file.